lib/server/push.api.js

```diff
@@ -36,6 +36,15 @@ Push.Configure = function(options) {
     // Add debug info
     if (Push.debug) console.log('Push.Configure', options);
 
+    // config collections for project
+    var prefix = options.dbCollectionPrefix || '';
+    var notificationsCollection =  prefix + '_raix_push_notifications';
+    Push.notifications = new Mongo.Collection(notificationsCollection);
```

Comment:
In this case, you should leave it on lib/
and with `Meteor.isServer` set the prefix on server, but in client you leave the collection as it is.